mail-manager/src/test/java/inc/softserve/EnvelopeServiceTest.java

```diff
@@ -18,16 +20,16 @@
 public class EnvelopeServiceTest extends Assert {
 
     private EnvelopeService service;
+    @Mock
     private UserDao usrDao;
+    @Mock
     private EnvelopeDao envDao;
+    @Mock
     private EnvelopeTools envTools;
```

Comment:
I think that we can keep those annotations on the same line with field declaration, although it clashes with regular Java Convention.